### PR TITLE
ref task 4758: grammar activity

### PR DESCRIPTION
**Pull request by nguyenducanhit**
https://edu-redmine.framgia.vn/issues/4795

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,6 +19,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".screen.grammar.GrammarActivity"
+            android:theme="@style/AppTheme.NoActionBar"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: check convention 
self close tag

File: app/src/main/java/com/framgia/toeic/screen/main/MainActivity.java [hunk 2]
```diff
@@ -13,6 +14,7 @@
 
 import com.framgia.toeic.R;
 import com.framgia.toeic.screen.base.BaseActivity;
+import com.framgia.toeic.screen.grammar.GrammarActivity;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/screen/main/MainActivity.java]: remove unuse import

File: app/src/main/java/com/framgia/toeic/screen/main/MainActivity.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.framgia.toeic.screen.main;
 
+import android.content.Intent;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/screen/main/MainActivity.java]: remove unuse import

**Comment by nguyenducanhit** [comment]
ready

**Pull request merged**